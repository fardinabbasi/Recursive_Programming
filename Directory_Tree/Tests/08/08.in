12 5
file1 0
folder1 0 
file2 2   
folder2 2 
folder3 4 
folder4 5
folder5 6
file3 7
file4 5
folder6 2
folder7 10
file5 11