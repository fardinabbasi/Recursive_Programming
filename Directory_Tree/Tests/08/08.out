|__file1
|__folder1
   |__file2
   |__folder2
   |  |__folder3
   |     |__folder4
   |     |  |__folder5
   |     |__file4
   |__folder6
      |__folder7
         |__file5